haskell/test/SMT/BaseDAO/Management.hs

```diff
@@ -0,0 +1,42 @@
+-- SPDX-FileCopyrightText: 2021 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module SMT.BaseDAO.Management
+  ( applyAcceptOwnership
+  , applyTransferOwnership
+  ) where
+
+import Universum
+
+import Lorentz hiding (not, cast, checkSignature)
+import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
+
+import Ligo.BaseDAO.Types
+import SMT.BaseDAO.Common
+import SMT.Types
+
+-- -- TODO: Not sure if operation can be checked in smt
```

Comment:
Well, not really (AFAIK we cannot get the operations list from the emulated nettest), but instead of checking the operations we can check the storage of other contracts.

To do so we need to use the same strategy that we use in unit tests with `consumerContract`, except that here the storage of such contract is stored in the whole `ModelState` as well and we need it to have an FA2 interface (in the case of the governance token).

Reply:
@pasqu4le I've updated most of the changes you suggested. Only this one is left, and I want clarify whether or not we want to test this. 

I'm not sure if this is worth it and it may complicate a lot of thing, so if you think this is necessary, do let me know.

Reply:
Yes, we need to test this,

Say there is a bug in the amount of money sent to a contract, with an operation, how could we know that this is happening without such a check?

It doesn't sound like much in `baseDAO` itself, but it's important considering `registryDAO` and `treasuyDAO`.

Reply:
Took me a while to implement this, I have added `SimpleOperation` which only supports entrypoint with `[FA2.TransferItem]` as parameter for now. I believe this should be enough for our use case.

I've also added the `CallCustom` entrypoints. The generator is improved a bit, but still have some more works to improve on it.